CONTRIBUTING.md

```diff
@@ -24,7 +48,8 @@ This is a rough outline of what a contributor's workflow looks like:
 - Make sure commit messages are in the proper format (see below).
 - Push changes in a topic branch to a personal fork of the repository.
 - Submit a pull request to operator-framework/operator-lifecycle-manager.
-- The PR must receive a LGTM from two maintainers found in the MAINTAINERS file.
+- The PR must receive a LGTM from a reviewer found in the OWNERS file.
```

Comment:
It appears that a new PR automatically suggests reviewers, but those reviewers don't seem to be obligated to provide a review. If someone provides a review, but does not provide a LGTM, then the issue would sit there forever. How would one ensure that a LGTM eventually gets provided so you can move on to the approval step?

Reply:
Also, what should we do when we make the suggested changes in a review? Should we click the "re-request review" button for the person that provided the review?

Reply:
In the section. `- Push changes in a topic branch to a personal fork of the repository.`, I also think it should be made clear the DCO requirement means you should use the --signoff flag when doing commits. If you don't do this then you have to go back and rebase your commits to get the signoff comments in place, which is a hassle.... the process to  do that is:

```bash
git rebase -i ID_OF_COMMIT_BEFORE_FIX
# change pick to edit for the commit(s) needing a signoff and save.
git commit --amend --no-edit --signoff
git rebase --continue
```

Reply:
Also I followed the guide here https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#the-code-review-process but this process also did not answer my questions above

Reply:
@jchunkins great questions!

>It appears that a new PR automatically suggests reviewers, but those reviewers don't seem to be obligated to provide a review. If someone provides a review, but does not provide a LGTM, then the issue would sit there forever. How would one ensure that a LGTM eventually gets provided so you can move on to the approval step?

There is no requirement for the suggested reviewers to review the PR, typically I would recommend pinging us using one of the mentioned communication channels. I will update the document to capture this step.

> Also, what should we do when we make the suggested changes in a review? Should we click the "re-request review" button for the person that provided the review?

Probably best to ping us again if all the requested changes have been made and the PR hasn't been re-reviewed. Will updated doc to reflect this.

> In the section. - Push changes in a topic branch to a personal fork of the repository., I also think it should be made clear the DCO requirement means you should use the --signoff flag when doing commits...

Great idea, will be added in the next update.